mentors/src/data_pipeline/transcript_adapter.py

```diff
@@ -0,0 +1,71 @@
+import os
+import pandas as pd
+
+"""
+This script will adapt the transcript.csv into questions_paraphrases_answers.csv
+The transcript.csv file is output by the data pre-processing stage of our data
+pipeline.  which is output by our data
+pre-processing stage
+"""
+DATA_ORDER = ["Topics", "Helpers", "Mentor", "Question", "text"]
+DATA_FOLDER = "data"
+TOPIC_MAP_FILE = os.path.join(DATA_FOLDER, "master_topic_map.csv")
+PARAPHRASE_MAP_FILE = os.path.join(DATA_FOLDER, "master_paraphrase_map.csv")
+TARGET_FILE = os.path.join(DATA_FOLDER, "questions_paraphrases_answers.csv")
+
+
+def build_qpa_data(mentor):
+    """
+    This function builds the questions_paraphrases_answers.csv file by leveraging
+    data from the transcript.csv file, the master_question_topic_map.csv, and the
+    master_question_paraphrase_map.csv
+    """
+    # Transcript
+    transcript_exists = True
+    session = 1
+
+    while transcript_exists:
+        filename = "data/recordings/{}/session{}/out/transcript.csv".format(
```

Comment:
we don't need to support python 2. Prefer f-strings unless there's the case really needs old style, e.g.

```
filename = f"data/recordings/{mentor}/session{session}/out/transcript.csv"
```

Reply:
Wow I never knew about that, awesome!